Integrations-NotTested/integrations-proofpoint_threat_response.yml

```diff
@@ -0,0 +1,365 @@
+description: 'Proofpoint builds lists for enforcement actions '
+detaileddescription: |-
+  In order to create API Key
+  Settings -> API Key -> Add api key
+configuration:
+- display: Server URL (e.g https://uxnysecthreat01.homebox.com)
```

Comment:
@bakatzir need to have a better default domain name,

Reply:
True, did it.